CONTRIBUTING.md

```diff
@@ -1,146 +1,107 @@
-  [GitHub Repository][github] as a Pull Request.
+```ts
+// add an example to examples/<your-example>.ts
 
-## Want a Doc Fix?
+#!/usr/bin/env -S npm run tsn -T
+…
+```
 
-If you want to help improve the docs in the helper library, it's a good idea to
-let others know what you're working on to minimize duplication of effort. Create
-a new issue (or comment on a related existing one) to let others know what
-you're working on.
+```sh
+$ chmod +x examples/<your-example>.ts
+# run the example against your api
+$ yarn tsn -T examples/<your-example>.ts
+```
 
-For large fixes, please build and test the documentation before submitting the
-PR to be sure you haven't accidentally introduced layout or formatting issues.
+## Using the repository from source
 
-## Submission Guidelines
+If you’d like to use the repository from source, you can either install from git or link to a cloned repository:
 
-### Submitting an Issue
+To install via git:
 
-Before you submit your issue search the archive, maybe your question was already
-answered.
+```sh
+$ npm install git+ssh://git@github.com:team-telnyx/telnyx-node.git
+```
 
-If your issue appears to be a bug, and hasn't been reported, open a new issue.
-Help us to maximize the effort we can spend fixing issues and adding new
-features by not reporting duplicate issues. Providing the following information
-will increase the chances of your issue being dealt with quickly:
+Alternatively, to link a local copy of the repo:
 
-- **Overview of the Issue** - if an error is being thrown a non-minified stack
-  trace helps
-- **Motivation for or Use Case** - explain why this is a bug for you
-- **`telnyx-node` Version(s)** - is it a regression?
-- **Operating System (if relevant)** - is this a problem with all systems or
-  only specific ones?
-- **Reproduce the Error** - provide an isolated code snippet or an unambiguous
-  set of steps.
-- **Related Issues** - has a similar issue been reported before?
-- **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point
-  to what might be causing the problem (line of code or commit)
+```sh
+# Clone
+$ git clone https://www.github.com/team-telnyx/telnyx-node
+$ cd telnyx-node
 
-**If you get help, help others. Good karma rules!**
+# With yarn
+$ yarn link
+$ cd ../my-package
+$ yarn link telnyx
 
-### Submitting a Pull Request
+# With pnpm
+$ pnpm link --global
+$ cd ../my-package
+$ pnpm link -—global telnyx
+```
 
-Before you submit your pull request consider the following guidelines:
+## Running tests
 
-- Search [GitHub][github] for an open or closed Pull Request that relates to
-  your submission. You don't want to duplicate effort.
-- Make your changes in a new git branch:
+Most tests require you to [set up a mock server](https://github.com/stoplightio/prism) against the OpenAPI spec to run the tests.
 
-```shell
-git checkout -b my-fix-branch master
+```sh
+$ npx prism mock path/to/your/openapi.yml
```

Comment:
can we update https://github.com/team-telnyx/telnyx-prism-mock?

Reply:
you can try swapping out in https://github.com/team-telnyx/telnyx-node/blob/next/scripts/mock#L24 and seeing if that works